cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -167,7 +168,10 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    if BZ(1608023, forced_streams=['5.9']).blocks:
+        time.sleep(35)
     view.save.click()
+    view.browser.refresh()
```

Comment:
Question: why we need this view.browser.refresh() here

Reply:
@rlbabyuk After saving creds, it won't reflect immediately and navigate to other page.